fluent-lts-apt-source/Rakefile

```diff
@@ -0,0 +1,146 @@
+# -*- ruby -*-
+
+require_relative "../lib/package-task"
+require 'rake/clean'
+require "time"
+
+# Debian
+CLEAN.include("apt/tmp")
+CLEAN.include("apt/build.sh")
+CLEAN.include("apt/env.sh")
+CLEAN.include("debian/tmp")
+CLOBBER.include("apt/repositories")
+
+class FluentdAptSourcePackageTask < PackageTask
```

Comment:
For the same reason, we don't need to separate the class name, right?

Reply:
Thanks, I've fixed it.

Reply:
Oh! We needed to fix this! OK!